### DISCUSSION TITLE
World file parsing

### DISCUSSION THREAD

**Discussion by JtotheThree** [mattyhall/rs-tiled#319] [Ideas]
Is there any interest in the crate being able to serialize .world files?

Is it just not a popular feature so it no one has added it yet? Or is this a feature in tiled that's not really well supported anymore?

**Comment by bjorn** [comment]
I think support for serializing .world files would be welcome in this crate. While the worlds feature of Tiled was initially added as just a means to show multiple maps together in Tiled, some people might have a use for it in their game, especially when manually editing the locations of maps in their world.

Is it something you'd be interested in adding?

**Reply by JtotheThree** [reply]
Yeah I'll get to work on it. We're looking at it as a possible chunking strategy for a Bevy crate downstream.
